test_suite/testSD.py

```diff
@@ -14,40 +19,68 @@ def findFastqFiles(directory, pattern):
+    return subprocess.check_output(
+        command, stderr=subprocess.STDOUT, shell=True)
+
+
+def file_compare(command, expected, returned):
+    # testing expected file output
     subprocess.check_output(command, stderr=subprocess.STDOUT, shell=True)
-    return filecmp.cmp('no_dup_interleaved.fastq', 'expected_interleaved.fastq') 
+    return filecmp.cmp(expected, returned)
 
-def duplicateReads():
-    command = "../super_deduper -1 fastqFiles/testCase_1X_R1.fastq -2 fastqFiles/testCase_1X_R1.fastq -N -F"
-    myString =  subprocess.check_output(command, stderr=subprocess.STDOUT, shell=True)
-    return myString
 
+def parse_fastq(filename):
+    # output the file to a dictionary
+    with open(filename) as f:
+        lines = f.readlines()
+    head = [item[:-1] for item in lines[0::4]]
+    read = [item[:-1] for item in lines[1::4]]
+    qual = [item[:-1] for item in lines[3::4]]
+    return {'Headers': head, 'Sequences': read, 'QScores': qual}
 
-class SuperDeduperTestCase(unittest.TestCase):
+
+class TestCase(unittest.TestCase):
+
+    def setUp(self):
+        """Setup any thing used in muliple tests"""
+        myR1file = " -1 fastqFiles/testCase_1X_R1.fastq "
+        myR2file = " -2 fastqFiles/testCase_1X_R2.fastq "
+        additFlags = "-N -F"
+        myShellCmd = "../super_deduper"
+        self.myCommand = myShellCmd + myR1file + myR2file + additFlags
 
     def test_find_fastq_files_recursively(self):
         """Should return all fastq files from the sub directories"""
-        self.assertEqual(findFastqFiles('fastqFiles', '*.fastq'),
-                         ['fastqFiles/testCase_1X_R1.fastq', 
-                          'fastqFiles/testCase_1X_R2.fastq', 
-                          'fastqFiles/testCase_2X_R1.fastq',
-                          'fastqFiles/testCase_2X_R2.fastq',
-                          'fastqFiles/testCase_3X_R1.fastq',
-                          'fastqFiles/testCase_3X_R2.fastq'])
-    
-    def test_duplicate_reads(self):
-        """Should return a tab delimited out put with 11 reads and 10 writes"""
-        self.assertEqual(duplicateReads(),
-        'Reads_Read\tReads_Written\tReads_Discarded\tSingletons\tDoubles\tThree_Plus\tDisqualifed_Reads\tReplacements_Called\tTotal_Time\n12\t11\t1\t10\t1\t0\t0\t0\t0\n')
-
-    def test_out_interleaved(self):
-		"""Should compare the output file to the expected_interleaved"""
-		self.assertTrue(interLeaved())
-
-        
-        
+        self.assertEqual(find_fastq_files('fastqFiles', '*.fastq'),
+                         ['fastqFiles/expected_R1.fastq',
+                          'fastqFiles/testCase_1X_R1.fastq',
+                          'fastqFiles/testCase_1X_R2.fastq'],
+                         "Unable to find test files")
+
+    def test_tab_output(self):
+        """Tests for 'Reads_Read' in the output"""
+        self.assertIn("Reads_Read\tReads_Written\tReads_Discarded\t"
+                      "Singletons\tDoubles\tThree_Plus\t"
```

Comment:
Hey @bioSandman. It looks great but I just changed up the format to add "Reads_Per_Second" so it is failing "Unexpected tab structure". It is on latest develop branch. Thank you!